.eslintrc.json

```diff
@@ -10,7 +10,13 @@
     "after": true,
     "before": true,
     "describe": true,
-    "it": true
+    "it": true,
```

Comment:
bitte keine nutzerspezifischen Settings pushen

Reply:
Die zugefügten globals ergeben sich aus der `base.js`, deren Funktionen global in der gesamten App verfügbar sind (die werden als non-basescripts in die `all.js` gebundelt). Es sind also defintiv keine nutzerspezifischen Settings. ;) 
![image](https://user-images.githubusercontent.com/7314829/42758937-80d0351a-8905-11e8-990b-f24eea7cba09.png)

Reply:
Ah I see, mein Fehler 👍